db/index.js

```diff
@@ -0,0 +1,50 @@
+//https://mongoosejs.com/docs/index.html
+const set = require('./complete.js');
```

Comment:
- Rename to `stubData`, or `data`, or `dataSet`, or similar. `set` implies this may be a "set" data structure.

- Also, rename the `complete.js` file so that one would know what is inside without having to open the file.